packages/amplify_datastore/example/lib/main.dart

```diff
@@ -5,11 +5,12 @@ library sample_app;
 
 import 'dart:async';
 
+import 'package:amplify_core/amplify_core.dart';
```

Comment:
I think we should import from `amplify_flutter` to ensure platform stuff can be correctly configured.

Reply:
Yes, it's required. `amplify_core` should only be imported directly in Dart-only environments.

Reply:
Didn't mean to include this file in this PR, will be removed 👍